app/javascript/react/components/TrailShowContainer.js

```diff
@@ -14,28 +15,34 @@ const TrailShowContainer = (props) => {
                 throw(error)
             }
             const fetchedTrail = await response.json()
-            setTrail(fetchedTrail)
+            setTrail(fetchedTrail.trail)
+            setReviews(fetchedTrail.trail.reviews)
         } catch(err) {
             console.error(`Error in fetch: ${err.message}`)
         }
     }
-
+    
     useEffect(() => {
         getTrail()
     }, [])
 
-    return (
-        <TrailShowTile
-            id={trail.id}
-            name={trail.name}
-            city={trail.city}
-            zip={trail.zip}
-            route_type={trail.route_type}
-            length={trail.length}
-            elevation={trail.elevation}
-            difficulty={trail.difficulty}
-            description={trail.description}
-        />
+    return ( 
+        <div>
+            <TrailShowTile
+                id={trail.id}
+                name={trail.name}
+                city={trail.city}
+                zip={trail.zip}
+                route_type={trail.route_type}
+                length={trail.length}
+                elevation={trail.elevation}
+                difficulty={trail.difficulty}
+                description={trail.description}
```

Comment:
in the future you may decide to just pass `trail={trail)` rather than calling on each property individually here